dev-python/ensurepip-pip/ensurepip-pip-25.1.1-r1.ebuild

```diff
@@ -64,6 +88,24 @@ python_prepare_all() {
 		mkdir tests/data/common_wheels/ || die
 		cp "${wheels[@]}" tests/data/common_wheels/ || die
 	fi
+
+	# Verify that we've covered licenses for all vendored packages
+	cd src/pip/_vendor || die
+	local packages=( */ [A-Za-z]*.py )
+	local packages_sorted=$(sort < <(printf "%s\n" "${packages[@]%/}") || die)
+	local have=$(sort < <(printf "%s\n" "${!VENDOR_LICENSES[@]}") || die)
+	local nl=$'\n'
+	if [[ ${packages_sorted} != ${have} ]]; then
+		eerror "VENDOR_LICENSES: ${have//${nl}/ }"
+		eerror "Packages found:  ${packages_sorted//${nl}/ }"
+		die "VENDOR_LICENSES outdated"
+	fi
+	local upstream_count=$(wc -l < vendor.txt || die)
+	if [[ ${#packages[@]} -ne ${upstream_count} ]]; then
+		eerror "VENDOR_LICENSES: ${#packages[@]}"
```

Comment:
Maybe replace `VENDOR_LICENSES:` by `Found packages:` or something, assuming this is meant to compare found files with vendor.txt, `VENDOR_LICENSES: *not* ${#VENDOR_LICENSES[@]}` was confusing me.